tests/management/test_finalize_xblockskill_tags.py

```diff
@@ -46,7 +47,6 @@ def test_finalize_xblockskill_tags_without_unverified_skills(self):
                 ]
```

Comment:
Please change the `call_command` here to not use the earlier options.

Reply:
Updated. Thanks!